### PR TITLE
cleanup setup script for mac

### PR DESCRIPTION
**Pull request by superpoussin22**
### Describe the Pull Request

```
Clean setup script for Macos 
Remove unneeded  check and tools
```

### Checklist for PR

- [X] Run MobSF unit tests and lint `tox -e lint,test`
- [X] Tested Working on Linux, Mac and Docker
- [X] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [X] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
if Ok need to clean documentation too
```

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -32,43 +32,6 @@ else
...
 
-# macOS Specific Checks
-if [[ $unamestr == 'Darwin' ]]; then
-    export ARCHFLAGS='-arch x86_64'
-    export LDFLAGS='-L/usr/local/opt/openssl/lib'
-    export CFLAGS='-I/usr/local/opt/openssl/include'
-    current_macos_version="$(sw_vers -productVersion | awk -F '.' '{print $1 "." $2}')"
-    major=$(echo "$current_macos_version" | cut -d'.' -f1)
-    # Check if xcode is installed
-    xcode-select -v
```
Comment (ajinabraham) [inline setup.sh:43]: xcode is not required for compiling any of the dependencies?

Comment (superpoussin22) [reply setup.sh:43]: Not on my Mac, will ask friends to test too

Comment (superpoussin22) [reply setup.sh:43]: when you install git commande line is installed , so I revert back the test , but remove all the rest

**Comment by ajinabraham** [comment]
@superpoussin22 Is this ready for review. More people are facing issues in latest mac. We need to rollout an interim fix.

**Comment by superpoussin22** [comment]
> @superpoussin22 Is this ready for review? More people are facing issues in latest mac. We need to rollout an interim fix.

yes @ajinabraham

**Pull request merged**